### DISCUSSION TITLE
Github, Instagram, Facebook, Twitter etc. Why are logo icons not available?

### DISCUSSION THREAD

**Discussion by ahmetcanisik** [tailwindlabs/heroicons#1176] [Ideas]
#### here are some famous brands I mentioned:
- Github
- Instagram
- Facebook
- Twitter
- Threads
- Linkedin
etc

**Comment by zefir-git** [comment]
Why burden Heroicons with brands/logos when https://simpleicons.org/ is specifically for this purpose
